cluster-state-service/handler/store/datastore.go

```diff
@@ -158,7 +181,17 @@ func (datastore etcdDataStore) stream(ctx context.Context, keyPrefix string, kvC
 			}
 			resetStreamIdleTimer(streamIdleTimer)
 			for _, ev := range event.Events {
-				kv := map[string]string{string(ev.Kv.Key): string(ev.Kv.Value)}
+				// Skip empty events, such as Etcd deletes.
```

Comment:
The problem is that the value returned tries to get deserialized into a task or container instance object in the calling method. When no value is returned, it couldn't be deserialized, and it was throwing an error and killing the stream. Not sure how we want to handle this. What should the stream send out on this scenario?